src/components/Notification/index.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { Box, styled, useThemeProps } from '@mui/material';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClose } from '@fortawesome/free-solid-svg-icons/faClose';
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons/faArrowUpRightFromSquare';
+import { shape } from '@balena/design-tokens';
+import { MUILinkWithTracking } from '../MUILinkWithTracking';
+import { IconButtonWithTracking } from '../IconButtonWithTracking';
+import { useEffect } from 'react';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+
+export interface NotificationProps {
+	campaignId: string;
+	children: React.ReactNode;
+	illustration?: React.ReactNode;
+	title: string;
+	linkHref: string;
+	linkLabel: string;
+	onClose?: (event: React.SyntheticEvent<any>) => void;
+}
+const NotificationRoot = styled(Box, {
+	name: 'Notification',
+	slot: 'root',
+})<{ ownerState: NotificationProps }>(({ theme, ownerState }) => ({
+	alignItems: 'center',
+	backgroundColor: 'white',
+	borderRadius: shape.border_radius.md.value + 'px',
+	boxShadow: '0px 0px 8px 0px rgba(35, 68, 94, 0.3)', // TODO add a design token for box shadows
```

Comment:
Couldn't we somehow use whatever MUI applies to dialogs?

Reply:
Ideally I would use a Paper component but I don't like the default MUI shadow. I will look at it separately and come back here to remove the random value.